src/main/java/com/discordbolt/api/command/HelpCommand.java

```diff
@@ -67,13 +65,4 @@ public static void helpCommand(CommandContext cc) {
         else
             cc.replyWith("No available commands.");
     }
-
-    @EventSubscriber
-    public void helpMention(MentionEvent e) {
```

Comment:
I want this to stay part of the command api, it is only possible to be triggered if you explicitly tag the bot and ask for the prefix, so I see no issues keeping it in. Normal bot use should not be invoked by tagging.

I would accept making this configurable with default on, but that should be done in a different PR.